### ISSUE TITLE
less html in content?

### ISSUE THREAD

**Issue by maelle** [apreshill/blogophonic#29]
I am wondering if about/index.html and elements/index.html might be misleading, could they work with Markdown & data templates?

**Comment by apreshill** [comment]
yes! Agreed. Here was my cryptic note-to-self on `about`: https://github.com/apreshill/blogophonic/issues/8

And elements I need to think about. It is showing what the CSS library can handle 🤔 which may be helpful at some level.

**Comment by maelle** [comment]
oooh right so users could delete elements. I suppose the issue can be closed then. :-)

**Comment by apreshill** [comment]
I'll reopen- want to consider how to tell people how to do 2 columns in an article
